### DISCUSSION TITLE
请求新建分支

### DISCUSSION THREAD

**Discussion by D-Sketon** [theme-shoka-x/hexo-theme-shokaX#57] [General]
我把anime精简了一下()

**Comment by zkz098** [comment]
这个是不是应该新建一个repo更合适()

**Reply by D-Sketon** [reply]
那岂不是应该建立一个组织()

**Reply by zkz098** [reply]
你怎么知道我想新建一个组织()

**Reply by D-Sketon** [reply]
不都这样的吗()

**Reply by zkz098** [reply]
我原本就准备新建一个组织，现在有理由了()
theme-shoka-x很快就来(

**Reply by D-Sketon** [reply]
好好好

**Reply by zkz098** [reply]
建好了()
目前正在迁移仓库，shokaX主仓库因为兼容性问题需要等下一个版本再迁移

**Reply by D-Sketon** [reply]
建议给组织加个icon
顺便加个好友吧()

**Reply by zkz098** [reply]
加个什么Icon，目前ShokaX似乎还没有LOGO()
哪个好友，指github follow吗()

**Reply by D-Sketon** [reply]
qq什么的？感觉github上交流不方便()

**Reply by zkz098** [reply]
坏了，我不用qq()
可以加个微信()

**Reply by D-Sketon** [reply]
也可以（

**Reply by zkz098** [reply]
微信号怎么发过去()
这玩意公开发有点离谱()

**Reply by D-Sketon** [reply]
之前不是有过邮箱吗()

**Reply by zkz098** [reply]
考虑将ShokaX-anime发布到npm上吗，jsd的网络质量十分离谱，走github可能不一定哪天就挂了

**Reply by D-Sketon** [reply]
要传吗，到时候怎么引用呢

**Reply by D-Sketon** [reply]
https://cdn.jsdelivr.net/npm/theme-shokax-anime@0.0.1/anime.shokax.min.js
这样引用吗

**Reply by zkz098** [reply]
所以说我应该用shokax.min.js还是min.js()

**Reply by D-Sketon** [reply]
都能用，只是shokax的更小点

**Comment by zkz098** [comment]
坏了，改成组织之后vercel不认了()，docs只能定期从主仓库同步了

**Comment by zkz098** [comment]
目前组织icon有候选的图片吗，如果没有我就从meme里选一张了()

**Reply by D-Sketon** [reply]
随便吧（）

**Reply by D-Sketon** [reply]
这头像有点逆天（）

**Reply by zkz098** [reply]
是的()

**Reply by zkz098** [reply]
但是我不是还没有update profile吗()

**Reply by zkz098** [reply]
原来不点也会更新()

**Reply by zkz098** [reply]
确实有点逆天()
换一种思路，找一个吉祥物，这样就有头像了()

**Reply by zkz098** [reply]
看不下去了()
紧急整了一张头像，先用这个
